### DISCUSSION TITLE
Any way to enable pinch to zoom on Android in fullscreen mode on Firefox?

### DISCUSSION THREAD

**Discussion by n00b12345** [gotson/komga#1757] [Q&A]
Is there any way to enable pinch to zoom in the fullscreen mode on Android (on Firefox)? Right now, pinch to zoom works perfectly as long as Komga is not in fullscreen mode. But once in fullscreen mode, pinch to zoom does not work. Enabling zooming on all websites using the accessibility settings on Firefox for Android also does not help. As you can imagine, small text becomes harder to read without zooming. (None of this applies to the desktop version of the web page, by the way, which works perfectly)

I know there are Android apps which work with Komga, but the web version is already so beautiful and minimalist in design, I really want to keep using it.

**Comment by gotson** [comment]
No, see #1596
